01-introduction.Rmd

```diff
@@ -2,68 +2,65 @@
+In the coming chapters, you'll explore examples of data science in education. But first, let's talk about why data science in education is not easily definable. 
 
-This imaginary building we call education also has rooms most residents never see---rooms where business and finance staff plan the most efficient use of limited funds. The transportation department plans bus routes across vast spaces. University administrators search for the best way to measure career readiness. Education consultants study how students perform on course work and even how they feel about class materials. 
+Being a data scientist in education is challenging because there is not a universal vision of what that means. For starters, education as a field is complex. If education were a building, it would be multi-storied with many rooms. There are privately and publicly funded schools. There are more than 18 possible grade levels. Students can learn alone or with others in a classroom. Students start school, leave school, finish school, and return to school. 
 
-There are a lot of ways one *could* do data science in education, but building consensus on ways one *should* do data science in education is just getting started. The "data science in education" community is still working out how it all fits together. 
+This imaginary building we call education also has rooms most never see -- rooms where business and finance staff plan the most efficient use of limited funds. The transportation department plans bus routes across vast spaces. University administrators search for the best way to measure career readiness. Education consultants study how students perform in coursework and how they feel about class materials. 
 
-And for someone just getting started, it can all seem very overwhelming. 
+There are many ways one *could* do data science in education, but consensus on ways one *should* do it is just starting. The community is still working out how it all fits together. 
 
-Even if we did have perfect clarity on the topic, there's still the issue of helping education systems learn to leverage these new analytical tools. In many education settings, school administrators and their staff may have never had someone around who deeply understands education, knows how to write code, and uses statistical techniques all at once, as data science in education could be defined [@conway2010data].
+Even then, there's still the issue of how education systems can best use data science tools. In many education settings, school staff have never worked with someone with this combination of skills -- education content knowledge, programming, and statistics [@conway2010data].
 
-## Making the path a little clearer
+## Making the path clearer
```

Comment:
I am wondering if somewhere here (or in the previous paragraph -- or earlier) we offer up a definition of educational data science. I like the definition of McFarland et al. (2021) -- an expansive definition that conceives of educational data science as "an umbrella for a range of new and often nontraditional quantitative methods (such as machine learning, network analysis, and natural language processing) applied to educational problems often using novel data" (p. 2). 

@article{mcfarland2021education,
  title={Education data science: Past, present, future},
  author={McFarland, Daniel A and Khanna, Saurabh and Domingue, Benjamin W and Pardos, Zachary A},
  journal={AERA Open},
  volume={7},
  pages={23328584211052055},
  year={2021},
  publisher={SAGE Publications Sage CA: Los Angeles, CA}
}

Reply:
I added this definition and reference. Thanks!